packages/amplify-provider-awscloudformation/src/resource-package/resource-packager.ts

```diff
@@ -57,7 +57,7 @@ export abstract class ResourcePackager {
 
   constructor(context: $TSContext, deployType: ResourceDeployType) {
     this.context = context;
-    this.elasticContainerZipFiles = ['custom-resource-pipeline-awaiter.zip', 'codepipeline-action-buildspec-generator-lambda.zip'];
+    this.elasticContainerZipFiles = ['custom-resource-pipeline-awaiter-18.zip', 'codepipeline-action-buildspec-generator-lambda.zip'];
```

Comment:
see: https://github.com/aws-amplify/amplify-category-api/pull/2732/files#r1699028311.

What is the number 18 referencing? What would cause this to change in the future? Should this be parameterized?

Reply:
looks like a ref to node18